firmware/quadruna/BMS/src/app/app_tractiveSystem.c

```diff
@@ -27,7 +27,7 @@ void app_tractiveSystem_broadcast()
 {
     const float ts_voltage = app_tractiveSystem_getVoltage();
     const float ts_current = app_tractiveSystem_getCurrent();
-    const float ts_power   = ts_voltage * ts_current;
+    const float ts_power   = ts_voltage * ts_current / 1000.0f;
```

Comment:
nit: lil somthin' like this
```suggestion
    #define W_TO_KW 1.0e-3f
    const float ts_power_kw   = ts_voltage * ts_current * W_TO_KW;
```